flox/aggregations.py

```diff
@@ -470,6 +470,19 @@ def _pick_second(*x):
 # median = Aggregation("median", chunk=None, combine=None, fill_value=None)
 # nanmedian = Aggregation("nanmedian", chunk=None, combine=None, fill_value=None)
 
+# Cumulatives:
+cumsum_ = Aggregation(
+    "cumsum",
+    chunk="cumsum",
+    combine="sum",
+    fill_value=0,
```

Comment:
fill_value and final_fill_value don't amke any sense?

Set `chunk=None` to raise an error with dask

Reply:
done